web/src/features/moreCast2/slices/parseModelsForStationsHelper.ts

```diff
@@ -0,0 +1,47 @@
+import { isNumber } from 'lodash'
+import { DateTime } from 'luxon'
+import { ModelsForStation, ModelValue } from 'api/modelAPI'
+import { ModelType } from 'api/nextCastAPI'
+import { NextCastForecastRow } from 'features/moreCast2/interfaces'
+
+// Convert the model predictions from the API to a format that can be used by the data grid
+export const parseModelsForStationsHelper = (stations: ModelsForStation[]) => {
+  const rows: NextCastForecastRow[] = []
+
+  stations.forEach(station => {
+    const station_code = station.station.code
+    const station_name = station.station.name
+    const model: ModelType = station.model_runs[0].model_run.abbreviation as ModelType
+    const values = station.model_runs[0].values
+    values.forEach((modelValue: ModelValue) => {
+      const row: NextCastForecastRow = {
+        id: Math.floor(Math.random() * 1000000), // TODO - Use a hash of the station name and date string?
```

Comment:
Is this ID only needed for the frontend DataGrid table?

Reply:
Yes, an ID is a requirement for the datagrid. Since we wouldn't need to do a lookup with the hash, it is probably worth pulling in the uuid package and just using a uuid for the id. What do you think?

Reply:
Yep that's what I was gonna suggest, I wasn't sure if we should just return that from the database lookup from API or not.

For now we can just do it on the frontend and change later if needed. Do we need to pull in the `uuid` package or can we use this OOTB?

https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID